bugsnag-plugin-android-ndk/src/androidTest/java/com/bugsnag/android/ndk/migrations/EventMigrationTest.kt

```diff
@@ -22,12 +23,18 @@ open class EventMigrationTest {
         }
     }
 
-    internal fun parseJSON(file: File): Map<*, *> {
-        return objectMapper.readValue(file, Map::class.java)
+    internal fun parseJSON(file: File): Map<String, Any> {
+        return deserialize(Files.readAllBytes(file.toPath()))
```

Comment:
```suggestion
        return deserialize(file.readBytes()))
```

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/java.io.-file/read-bytes.html

Reply:
:pray: